### DISCUSSION TITLE
Nothing happens on pressing INSERT

### DISCUSSION THREAD

**Discussion by anitsingh367** [danielkrupinski/Osiris#2773] [Q&A]
I did everything as per the steps but nothing really happened when I pressed the Insert Key in the game.

**Comment by linuxer9** [comment]
what injector did u use?

**Reply by anitsingh367** [reply]
https://github.com/Arvanaghi/Windows-DLL-Injector
this is the injector I used

**Reply by linuxer9** [reply]
this injector will not work, you have to use an injector that bypasses the anti injection protection
try randoinjector
https://github.com/martinjanas/RandoInjector
